Aspen-Discovery/aspen-discovery
# Show summon search tools

Test Plan:

1. Before applying patch, set up Summon and carry out a search.
2. Notice that there is no checkbox to hide covers or option to switch between list and covers view at the top of the page.
3. Repeat the above with an Ebsco EDS search.
4. Apply the patch.
5. Notice that when you carry out a Summon Search now, the search tools are at the top of the screen as they are in Ebsco EDS and checking the hide covers box refreshes the page and returns results without the covers.
